netlify.toml

```diff
@@ -1,36 +1,33 @@
 [Settings]
-# Added automatically by the Netlify CLI. It has no effect during normal 
-# Git-backed deploys.
-ID = "14115dc9-bace-418d-8120-2a5b4d9a43ce"
```

Comment:
On purpose?

Reply:
Kinda, doesn't this need a new id and site on netlify first?

Reply:
No, I will just rename the current staging one - No worries!